package.json

```
@@ -6,8 +6,6 @@
   "bugs": "https://github.com/oclif/plugin-plugins/issues",
   "dependencies": {
     "@oclif/color": "^0.x",
```

Missing `@oclif/core`?
```suggestion
    "@oclif/color": "^0.x",
    "@oclif/core": "^0.5.39",
```